### PR TITLE
docs: update docs for the new naming convention

### PR DESCRIPTION
**Pull request by agnes512**
- update docs for the new naming convention
- Add naming convention section in CLI main page

resolves https://github.com/strongloop/loopback-next/issues/2644

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Command-line-interface.md [hunk 1]
```diff
@@ -25,3 +25,15 @@ npm install -g @loopback/cli
...
 
 {% include_relative tables/lb4-artifact-commands.html %}
+
+## Naming Convention
+
+LoopBack 4 uses different convention for naming classes, variables, and files.
+
+- Class name: `PascalCase`.
+- File name: `kebabCase`.
+- Variable name, `camelCase`.
```
Comment (raymondfeng) [inline docs/site/Command-line-interface.md]: It should be `Variable name:`

Comment (agnes512) [reply docs/site/Command-line-interface.md]: fixed

File: docs/site/Command-line-interface.md [hunk 2]
```diff
@@ -25,3 +25,15 @@ npm install -g @loopback/cli
 ## Generating LoopBack artifacts
 
 {% include_relative tables/lb4-artifact-commands.html %}
+
+## Naming Convention
+
+LoopBack 4 uses different convention for naming classes, variables, and files.
+
+- Class name: `PascalCase`.
+- File name: `kebabCase`.
```
Comment (bajtos) [inline docs/site/Command-line-interface.md]: I think the example should be `kebab-case`? The current string is in camel case.

File: docs/site/tutorials/soap-calculator/soap-calculator-tutorial-add-controller.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ lb4 controller calculator
 Next, select the **Empty Controller**.
 
 ```sh
+Controller Caculator will be created in src/controllers/caculator.controller.ts
```
Comment (nabdelgadir) [inline docs/site/tutorials/soap-calculator/soap-calculator-tutorial-add-controller.md]: typo: `caculator` -> `calculator`

File: docs/site/Command-line-interface.md (hunk 2 above)
Comment (agnes512) [reply docs/site/Command-line-interface.md]: true, I will fix it in a small PR real quick

**Pull request merged**